vaadin-spring/src/main/java/com/vaadin/flow/spring/router/SpringRouteNotFoundError.java

```diff
@@ -0,0 +1,28 @@
+package com.vaadin.flow.spring.router;
+
+import com.vaadin.flow.component.Html;
+import com.vaadin.flow.router.BeforeEnterEvent;
+import com.vaadin.flow.router.ErrorParameter;
+import com.vaadin.flow.router.NotFoundException;
+import com.vaadin.flow.router.RouteNotFoundError;
+
+public class SpringRouteNotFoundError extends RouteNotFoundError {
```

Comment:
Do I understand correctly that this message will be shown by default when the route not found error occurs?

Reply:
It should override the RouteNotFoundError.

Reply:
Yes, that's correct. It will be picked over the other default one from Flow `RouteNotFoundError`.

Reply:
Ok, understood, thanks!

Reply:
Just one more clarification: is that correct that the message from `RouteNotFoundError` would be overridden? What if we would have a route error just because some another problem than that one with Spring bug?

Reply:
the int retval = super.setError... adds the original content

Reply:
Now I see, thanks!